hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/MockSCMHAManager.java

```diff
@@ -92,11 +92,20 @@ public void shutdown() throws IOException {
     ratisServer.stop();
   }
 
-  private class MockRatisServer implements SCMRatisServer {
+  /**
+   * Mock RatisServer implementation for testing.
```

Comment:
Hey Rui, changes in MockSCMHAManager.java may not be needed, since MockRatisServer is an inner class of MockSCMHAManager,